scripts/scripts.js

```diff
@@ -12,6 +12,7 @@ import {
   loadBlocks,
   loadCSS,
 } from './lib-franklin.js';
+import { preDecorateEmbed } from './modules/embed.js';
```

Comment:
The `preDecorateEmbed` should stay in the scripts.js otherwise loading the additional module delays the dom content loaded until it got loaded. 

All other functionality may be in modules, but that actually only makes sense when the modules are imported conditionally (meaning for example only when there are embeds or twitter feeds on the page). Currently the additional network requests delay the execution of the `delayed.js`.

![Screenshot 2022-11-23 at 10 41 05](https://user-images.githubusercontent.com/558094/203514625-a7604516-b8cd-4483-9a7f-05fe93285ede.png)

Reply:
done